adafruit_connection_manager.py

```diff
@@ -203,6 +204,18 @@ def _get_key_for_socket(self, socket):
         except StopIteration:
             return None
 
+    @property
+    def open_sockets(self) -> int:
+        """Get the count of open sockets"""
+        return len(self._open_sockets)
```

Comment:
This name is ambiguous, since it could mean returning a collection of open sockets, instead of a count. `open_sockets_count` or `num_open_sockets` would be clearer.

What is the use case for having this available to the user?